modules/nf-core/pretextsnapshot/main.nf

```diff
@@ -0,0 +1,36 @@
+process PRETEXTSNAPSHOT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::pretextsnapshot=0.0.4" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pretextsnapshot:0.0.4--h7d875b9_0':
+        'quay.io/biocontainers/pretextsnapshot:0.0.4--h7d875b9_0' }"
+
+    input:
+    tuple val(meta), path(pretext_map)
+
+    output:
+    tuple val(meta), path('*.{jpeg,png,bmp}', glob: true), emit: image
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def outdir = task.ext.outdir ?: '.'
+    """
+    PretextSnapshot \\
+        $args \\
+        -m $pretext_map \\
+        --prefix $prefix \\
+        -o $outdir
```

Comment:
Can you use the long form if there is one ?
> Mandatory command line arguments MUST be specified in long form where possible. https://nf-co.re/docs/contributing/modules#general

Reply:
:scream: I always forget that one! Thanks for the reminder @muffato :sweat_smile: , and applies to `-m` too

In addition, why not just `${prefix}/` for outdir rather an having an additional non-standard `task.ext` (even if it is redundant with `--prefix`, otherwise I would just specify `.` instead)?

Reply:
Yes, I rarely see `task.ext.outdir` too. I think it's superfluous here, and you can probably just use `$prefix`